### ISSUE TITLE
Epochs not shown

### ISSUE THREAD

**Issue by Lia0107** [ultralytics/ultralytics#25952]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

Hi, currently I'm training yolo11s-seg and i realised it is not the same as previous. My command was always been this. But idk why now when training, it shows these and all data seems doesn't fit the column?
!yolo task=segment mode=train model=yolo11s-seg.pt data=/content/dataset/ornamental-chili-17/data.yaml epochs=100 imgsz=640 patience = 20

<img width="1219" height="666" alt="Image" src="https://github.com/user-attachments/assets/9cd332f3-47a5-48cb-91bf-cf5f5ab6caca" />

### Environment

Ultralytics 8.4.130 🚀 Python-3.13.15 torch-2.11.0+cu128 CUDA:0 (Tesla T4, 14913MiB)

### Minimal Reproducible Example

import os
import glob
from IPython.display import Image, display
from IPython import display
display.clear_output()

!pip install ultralytics
import ultralytics
ultralytics.checks()
!pip install roboflow
(import dataset from roboflow)
and use this to train
!yolo task=segment mode=train model=yolo11s-seg.pt data=ur_dataset/data.yaml epochs=100 imgsz=640 patience = 20

### Are you willing to submit a PR?

- [ ] #25953

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Lia0107, thank you for reporting the malformed training output and missing epoch display during YOLO11 segmentation training. This is an automated response, and an Ultralytics engineer will assist soon.

Please provide the exact console output as text, along with whether the issue occurs in a fresh runtime after upgrading to the latest `ultralytics` package. A [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example) will help us investigate the formatting and logging behavior.

**Comment by pderrenger** [comment]
Please restart the Colab runtime after upgrading `ultralytics`, then rerun with no spaces around `=`: `!yolo segment train model=yolo11s-seg.pt data=/content/dataset/ornamental-chili-17/data.yaml epochs=100 imgsz=640 patience=20`. If the epoch column remains misaligned, please paste the complete console output as text so we can reproduce the notebook formatting issue.

**Comment by Lia0107** [comment]
yes currently still the same i followed what you said, no space between = but still exist the header problem. Lack of logger metrics.
!yolo task=segment mode=train model=yolo11s-seg.pt data=/content/dataset/ornamental-chili-18/data.yaml epochs=100 imgsz=640 patience=20




Ultralytics 8.4.130 🚀 Python-3.13.15 torch-2.11.0+cu128 CUDA:0 (Tesla T4, 14913MiB)
engine/trainer: agnostic_nms=False, amp=True, angle=1.0, augment=False, auto_augment=randaugment, batch=16, bgr=0.0, box=7.5, cache=False, cfg=None, channels_last=False, classes=None, close_mosaic=10, cls=0.5, cls_pw=0.0, cls_remap=True, compile=False, conf=None, copy_paste=0.0, copy_paste_mode=flip, cos_lr=False, cutmix=0.0, data=/content/dataset/ornamental-chili-18/data.yaml, degrees=0.0, deterministic=True, device=, dfl=1.5, dgrad=0.5, dis=6.0, distill_model=None, dlam=1.0, dlog=1.0, dnn=False, dropout=0.0, dynamic=False, embed=None, end2end=None, epochs=100, erasing=0.4, exist_ok=False, fliplr=0.5, flipud=0.0, format=torchscript, fraction=1.0, freeze=None, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, imgsz=640, iou=0.7, keras=False, kobj=1.0, line_width=None, lr0=0.01, lrf=0.01, mask_ratio=4, max_det=300, mixup=0.0, mode=train, model=yolo11s-seg.pt, momentum=0.937, mosaic=1.0, multi_scale=0.0, name=train, nbs=64, nms=False, opset=None, optimize=False, optimizer=auto, overlap_mask=True, patience=20, perspective=0.0, plots=True, pose=12.0, pretrained=True, profile=False, project=None, quantize=None, rect=False, resume=False, retina_masks=False, rle=1.0, save=True, save_conf=False, save_crop=False, save_dir=/content/dataset/ornamental-chili-18/runs/segment/train, save_frames=False, save_json=False, save_period=-1, save_txt=False, scale=0.5, seed=0, shear=0.0, show=False, show_boxes=True, show_conf=True, show_labels=True, simplify=True, single_cls=False, source=None, split=val, stream_buffer=False, task=segment, time=None, tracker=tracktrack.yaml, translate=0.1, val=True, verbose=True, vid_stride=1, visualize=False, warmup_bias_lr=0.1, warmup_epochs=3.0, warmup_momentum=0.8, weight_decay=0.0005, workers=8, workspace=None
Overriding model.yaml nc=80 with nc=2

                   from  n    params  module                                       arguments                     
  0                  -1  1       928  ultralytics.nn.modules.conv.Conv             [3, 32, 3, 2]                 
  1                  -1  1     18560  ultralytics.nn.modules.conv.Conv             [32, 64, 3, 2]                
  2                  -1  1     26080  ultralytics.nn.modules.block.C3k2            [64, 128, 1, False, 0.25]     
  3                  -1  1    147712  ultralytics.nn.modules.conv.Conv             [128, 128, 3, 2]              
  4                  -1  1    103360  ultralytics.nn.modules.block.C3k2            [128, 256, 1, False, 0.25]    
  5                  -1  1    590336  ultralytics.nn.modules.conv.Conv             [256, 256, 3, 2]              
  6                  -1  1    346112  ultralytics.nn.modules.block.C3k2            [256, 256, 1, True]           
  7                  -1  1   1180672  ultralytics.nn.modules.conv.Conv             [256, 512, 3, 2]              
  8                  -1  1   1380352  ultralytics.nn.modules.block.C3k2            [512, 512, 1, True]           
  9                  -1  1    656896  ultralytics.nn.modules.block.SPPF            [512, 512, 5]                 
 10                  -1  1    990976  ultralytics.nn.modules.block.C2PSA           [512, 512, 1]                 
 11                  -1  1         0  torch.nn.modules.upsampling.Upsample         [None, 2, 'nearest']          
 12             [-1, 6]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 13                  -1  1    443776  ultralytics.nn.modules.block.C3k2            [768, 256, 1, False]          
 14                  -1  1         0  torch.nn.modules.upsampling.Upsample         [None, 2, 'nearest']          
 15             [-1, 4]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 16                  -1  1    127680  ultralytics.nn.modules.block.C3k2            [512, 128, 1, False]          
 17                  -1  1    147712  ultralytics.nn.modules.conv.Conv             [128, 128, 3, 2]              
 18            [-1, 13]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 19                  -1  1    345472  ultralytics.nn.modules.block.C3k2            [384, 256, 1, False]          
 20                  -1  1    590336  ultralytics.nn.modules.conv.Conv             [256, 256, 3, 2]              
 21            [-1, 10]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 22                  -1  1   1511424  ultralytics.nn.modules.block.C3k2            [768, 512, 1, True]           
 23        [16, 19, 22]  1   1474678  ultralytics.nn.modules.head.Segment          [2, 32, 128, 16, None, [128, 256, 512]]
YOLO11s-seg summary: 204 layers, 10,083,062 parameters, 10,083,046 gradients, 33.2 GFLOPs

Transferred 555/561 items from pretrained weights
Freezing layer 'model.23.dfl.conv.weight'
AMP: running Automatic Mixed Precision (AMP) checks...
AMP: downloading yolo26n.pt for AMP checks (one-time, not used for training)...

AMP: checks passed ✅
train: Fast image access ✅ (ping: 0.0±0.0 ms, read: 1677.7±789.8 MB/s, size: 55.2 KB)

train: New cache created: /content/dataset/ornamental-chili-18/train/labels.cache
albumentations: Blur(p=0.01, blur_limit=(3, 7)), MedianBlur(p=0.01, blur_limit=(3, 7)), ToGray(p=0.01, method='weighted_average', num_output_channels=3), CLAHE(p=0.01, clip_limit=(1.0, 4.0), tile_grid_size=(8, 8))
val: Fast image access ✅ (ping: 0.0±0.0 ms, read: 753.5±640.1 MB/s, size: 59.7 KB)

val: New cache created: /content/dataset/ornamental-chili-18/valid/labels.cache
optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... 
optimizer: AdamW(lr=0.001667, momentum=0.9) with parameter groups 90 weight(decay=0.0), 101 weight(decay=0.0005), 100 bias(decay=0.0)
Plotting labels to /content/dataset/ornamental-chili-18/runs/segment/train/labels.jpg... 
Using 1328 train, 166 val images for fraction=1.0 at imgsz=640
Using 2 dataloader workers
Logging results to /content/dataset/ornamental-chili-18/runs/segment/train
Starting training for 100 epochs...

      Epoch    GPU_mem   box_loss   seg_loss   cls_loss   dfl_loss   sem_loss  Instances       Size


                   all        166        430      0.597      0.473      0.515       0.35      0.599      0.474      0.531      0.365

**Comment by Y-T-G** [comment]
@Lia0107 Thanks for reporting. It's a bug.

**Comment by UltralyticsAssistant** [comment]
A potential fix is now available in the [merged pull request](https://github.com/ultralytics/ultralytics/pull/25958) by @Y-T-G.

Updated `TQDM` output fitting to handle zero-width terminals and notebook panes correctly, while marking terminal truncation with an ellipsis.

Please test the merged change using this repository's documented workflow. If the issue persists, share the updated behavior and any new diagnostic details. Thank you for reporting it! 🙏
